CHANGES.rst

```diff
@@ -222,6 +222,9 @@ astropy.io.registry
 astropy.io.votable
 ^^^^^^^^^^^^^^^^^^
 
+- Block floating-point columns from using repr format when converted to Table
```

Comment:
please move this to section 2.0.12

Reply:
Now moved to section 2.0.13 since I didn't get this resolved for 2.0.12.